### PR TITLE
Add Avril 14th rpc/grpc/api

### PR DESCRIPTION
**Pull request by stefanopier**

### CODE REVIEW

**Comment by reversesigh** [comment]
@stefanopier can you resolve the conflict please? ill merge afterward.

File: celestia/chain.json [hunk 1]
```diff
@@ -27,7 +27,7 @@
       {
         "denom": "utia"
       }
-    ]
+    ]https://github.com/cosmos/chain-registry/pull/3767/conflict?name=celestia%252Fchain.json&ancestor_oid=33bd2b78fc39338fae61d244e171f9da5ba1e43a&base_oid=934b76c86b7bc749643a07fe6ef81c9d28a4c88b&head_oid=61573e38ac6168e02e81447e78a53661a1f16969
```
Comment (JeremyParish69) [inline celestia/chain.json:30]: What's this doing here?

Comment (stefanopier) [reply celestia/chain.json]: just removed..

Comment (stefanopier) [reply celestia/chain.json]: let me know if you want me to update this fork, it's getting a little behind..

**Pull request merged**